charts/ccd-data-store-api/values.yaml

```diff
@@ -44,8 +44,6 @@ java:
     ELASTIC_SEARCH_NODES_DISCOVERY_FREQUENCY_MILLIS: '5000'
     ELASTIC_SEARCH_NODES_DISCOVERY_FILTER: '_all'
 
```

Comment:
do we have to bump up the chart version?

Reply:
no need it'll become unused environment variable and will be gone in the next version.

Reply:
👍